packages/asc-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -43,38 +43,28 @@ export const Tabs: React.FC<Tabs> = ({ label, children, ...otherProps }) => {
 
   return (
     <>
-      <TabList
-        aria-label={label}
-        onKeyDown={keyDown}
-        ref={tabListRef}
-        {...otherProps}
-      >
-        {children.map(
-          ({ props: { id, label: childLabel, ...otherChildProps } }) => {
```

Comment:
Why did you refactor this? Destructuring the props is more readable, in my opinion. And why did you remove `otherChildProps`? The idea was to enable the user to set props (such as `style`) on `Tab`, in order to style `TabButton`.